src/CMakeCPPConfig.txt

```diff
@@ -73,6 +73,14 @@ include_directories( SYSTEM ${CMAKE_HOME_DIRECTORY}/main/resources/lib/spdlog/in
 include_directories( SYSTEM ${CMAKE_HOME_DIRECTORY}/main/resources/lib/trng-4.15/include )
 #set( LIBS ${LIBS} trng )
 
+
+#----------------------------------------------------------------------------
+# Awesomium web view visualizer
+#----------------------------------------------------------------------------
+find_package (Awesomium REQUIRED)
```

Comment:
This is a stupid nitpick, but can you change the whitespace around these braces to match Stride's CMake code style?